src/learning-catalogue/model/course.ts

```diff
@@ -0,0 +1,76 @@
+import {Module} from './module'
+
+export class Course {
+	private _id: string
+	private _title: string
+	private _shortDescription: string
+	private _description: string
+	private _duration: number
+	private _learningOutcomes: string
+	private _price: number | 'Free'
+	private _modules: Module[]
+
+	get id(): string {
+		return this._id
+	}
+
+	set id(value: string) {
+		this._id = value
+	}
```

Comment:
As previous comment on functions we're implementing - is it OK to change a Course's ID? This is set by the API, here we're saying it's OK for the front end to set it as well.

Reply:
I'm not sure about this. I don't think having a setter necessarily means it's ok to change the id. It depends how the object is constructed. I'm happy to make the object immutable but I thought this might be overkill.